db-services/db-palette-service/src/main/java/com/qreal/wmp/db/palette/config/Dao.java

```diff
@@ -0,0 +1,35 @@
+package com.qreal.wmp.db.palette.config;
+
+import com.qreal.wmp.db.palette.dao.PaletteDao;
+import com.qreal.wmp.db.palette.dao.PaletteDaoImpl;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import static org.mockito.Mockito.mock;
```

Comment:
We may have problems with Spring contexts initalizing if we will move part of config to test folder. Nevertheless, moving tests config to test folder seems reasonable. If movement here will be successful, need to do it in all other db-services.